src/main/java/core/basesyntax/strategy/TransactionStrategy.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.strategy;
+
+public interface TransactionStrategy {
+    int balanceUpdater(int balance, String quantity);
+}
```

Comment:
Task of strategy pattern is to provide proper implementation of some interface depending on input
so it should be something like
```suggestion
public interface TransactionStrategy {
    TransactionHandler getHandler(FruitTransaction transaction);
}
```